release_notes.md

```diff
@@ -2,10 +2,15 @@
 
 ## Module Enhancements
 
-- Added a new module named **CVEs** in the *Vulnerability Management* navigation menu
+- Added new fields in **CVEs** module:
+    - `Date Added To KEV Catalog`
+    - `Response Due Date`
+    - `Vulnerability Name`
+    - `Action`
+    >**Note**: This field displays if the value for `Exists In KEV Catalog` is `Yes`
 
-- Added following new picklists related to the **CVEs** module
+- Modified field `Is In KEV Catalog` to `Exists In KEV Catalog`
 
-    - `CVEVulnerabilityStatus`
+- Changed detail view of CVEs Module:
```

Comment:
Suggestion: Updated detailed view of CVEs Module to accommodate newly added fields

Reply:
incorporated